simulator/examples/image-bmp.rs

```diff
@@ -10,10 +12,13 @@ use tinybmp::Bmp;
 fn main() -> Result<(), core::convert::Infallible> {
     let mut display: SimulatorDisplay<Rgb565> = SimulatorDisplay::new(Size::new(128, 128));
 
+    // Load the BMP image
     let bmp = Bmp::from_slice(include_bytes!("./assets/rust-pride.bmp")).unwrap();
 
+    // Create a new embedded-graphics Image, wrapping the BMP which provides pixel data
     let image: Image<Bmp, Rgb565> = Image::new(&bmp, Point::zero());
 
+    // Translate the image down and to the right by 32px, then display it
     image.translate(Point::new(32, 32)).draw(&mut display)?;
```

Comment:
The translation could be removed by changing the `Image::new` argument above. This would be the default way of creating an `Image` at some specific position.